README.md

```
@@ -1,6 +1,8 @@
 # ESPHome Tesla BLE
 
-This project lets you use an ESP32 device to manage charging a Tesla vehicle over BLE, using the [yoziru/tesla-ble](http://github.com/yoziru/tesla-ble) library.
+Forked from Forked from [yoziru/tesla-ble](https://github.com/yoziru/esphome-tesla-ble) 
```

Markdownlint (0.37.0)

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)